test/test_pipeline/test_tabular_regression.py

```diff
@@ -144,7 +150,10 @@ def test_default_configuration(self, fit_dictionary_tabular, is_small_preprocess
         pipeline = TabularRegressionPipeline(
             dataset_properties=fit_dictionary_tabular['dataset_properties'])
 
-        pipeline.fit(fit_dictionary_tabular)
+        with unittest.mock.patch.object(pipeline.named_steps['trainer'].choice, 'train_epoch') \
+                as patch_train:
+            patch_train.return_value = 1, {}
+            pipeline.fit(fit_dictionary_tabular)
```

Comment:
What is the intention behind L153-155?

Reply:
are we are testing if a pipeline can be fit, and we have a separate test for testing training, I replaced the training steps with a mock patch object to make it faster

Reply:
I checked the corresponding parts (base_trainer.py and base_trainer_choice.py) and it seems no problem.
Thanks.